### ISSUE TITLE
Missing alternative text for images

### ISSUE THREAD

**Issue by shiffman** [shiffman/shiffman.net#64]
There are many instances of alt text missing for images.

<img width="341" alt="screen shot 2018-10-19 at 2 28 41 pm" src="https://user-images.githubusercontent.com/191758/47236837-4bd63d00-d3ab-11e8-8dca-3ef46e4b917e.png">

On: https://shiffman.net/a2z/

**Comment by phucduong86** [comment]
On https://shiffman.net, need alt text for all video thumbnails
<img width="388" alt="home daniel shiffman 2018-11-06 21-17-17" src="https://user-images.githubusercontent.com/21692768/48106423-601e9480-e209-11e8-8a7e-68af048a5a53.png">

**Comment by shiffman** [comment]
@phucduong86 Can this be closed as your work and issues are more comprehensive and this is in progress?

**Comment by phucduong86** [comment]
@shiffman  Yes we can thanks!
